### DISCUSSION TITLE
very hard steps to get token stuff for onedrive..... ;(

### DISCUSSION THREAD

**Discussion by AmirulAndalib** [spencerwooo/onedrive-vercel-index#30] [Q&A] [answered]
hi there
# can you please provide use tutorial for getting refresh token access token for OD correctly i tried many times it doesnt work ....please help

**Comment by People-11** [comment, marked as answer]
> hi there
> 
> # can you please provide use tutorial for getting refresh token access token for OD correctly i tried many times it doesnt work ....please help

1. Install Node.js on your computer
2.  Run npx @beetcb/ms-graph-cli in the terminal or command line.
3.  Follow the steps to get the token

**Comment by hamasharreef** [comment]
easy way

https://duplicacy.com/one_start
